apps/experiments/export.py

```diff
@@ -93,3 +94,21 @@ def filtered_export_to_csv(experiment, sessions_queryset):
             ]
             writer.writerow(row)
     return csv_in_memory
+
+
+def _get_trace_info_for_export(message):
```

Comment:
Nit: Rename to `_get_trace_id_for_export` since this returns not the full trace info, but the `trace_id`.